client/my-sites/email/email-management/gsuite-users-card/style.scss

```diff
@@ -37,54 +37,54 @@
 
 .gsuite-users-card__foldable-card {
 	margin-bottom: 0;
+}
 
-	.gsuite-users-card__foldable-card-header-icon {
```

Comment:
I have made no changes to that file except removing unnecessary indentation - as recommended [here](https://github.com/Automattic/wp-calypso/blob/trunk/docs/components.md#syntax).